### PR TITLE
Ensure no dupe repos by accounting for owner

### PR DESCRIPTION
**Pull request by lhorie**

### CODE REVIEW

File: src/packageUtils.js [hunk 1]
```diff
@@ -210,36 +218,40 @@ async function installBatchedPackages(batches) {
...
               pkg.dependents[k]
-            }/node_modules/${pkg.name}`);
+            }/node_modules/${pkg.getPath()}`);
           } else {
             // Otherwise copy only the package files
             shelljs.exec(
-              `mkdir -p packages/${pkg.dependents[k]}/node_modules/${pkg.name}`
+              `mkdir -p packages/${
+                pkg.dependents[k]
+              }/node_modules/${pkg.getPath()}`
```
Comment (KevinGrandon) [inline src/packageUtils.js]: Ah, this part looks suspicious. I believe we would need to compute the right path based off of the package namespaced file.

File: src/packageUtils.js [hunk 2]
```diff
@@ -85,170 +86,188 @@ class Package {
...
+              // Otherwise copy only the package files
+              shelljs.exec(
+                `mkdir -p ${this.dir}/${
+                  pkg.dependents[k]
+                }/node_modules/${pkg.getPath()}`
+              );
+              ['package.json', ...pkg.files].forEach(file => {
+                const copyTo = `${this.dir}/${
+                  pkg.dependents[k]
+                }/node_modules/${pkg.getPath()}/${file}`;
```
Comment (KevinGrandon) [inline src/packageUtils.js]: Ah, this part looks suspicious. I believe we would need to compute the right path based off of the package namespaced file.

Comment (lhorie) [reply src/packageUtils.js]: Ok let me look into it

Comment (lhorie) [reply src/packageUtils.js]: I've updated the code and found some other places where wonky things were happening. Added some tests as well

**Comment by KevinGrandon** [comment]
Going to land this so I can iterate in master on some other release verification work without fear of causing merge conflicts.

**Comment by KevinGrandon** [comment]
!merge

**Pull request merged**